spec/models/bottleneck_event_spec.rb

```diff
@@ -76,4 +76,37 @@
       expect(query).to match(/resource_type = 'EmsCluster'.*resource_type = 'Host'/)
     end
   end
+
+  describe ".last_created_on" do
+    context "returns the last created_on date" do
+      def expect_last_created_on_to_eq_resource_last_created_on(resource)
+        bottleneck_event = BottleneckEvent.create!(:resource => resource)
+        expect(described_class.last_created_on(resource)).to eq(bottleneck_event.reload.created_on)
+      end
```

Comment:
@Fryguy what do you think of this way of DRYing up this spec? I'm not sure what the consensus is around here :)